### ISSUE TITLE
How do I use it for Electron

### ISSUE THREAD

**Issue by pavelfeldman** [microsoft/vscode-pwa#103]
I used to use pwa debugger for my Electron app with the following config:

```json
{
      "type": "pwa",
      "request": "launch",
      "name": "Electron App",
      "command": "npm run electron -- --remote-debugging-port=9222",
      "remoteDebuggingPort": "9222"
    },
```

Is this supported? How do I configure it?

**Comment by connor4312** [comment]
This should work for you in the next nightly (prev didn't allow 'server's in attach, which was silly https://github.com/microsoft/vscode-pwa/commit/e9d7a563aa5d9e1a2bfaf82e07717a4c59c92f02):

```js
{
  "type": "pwa-chrome",
  "request": "attach",
  "name": "Electron App",
  "port": 9222,
  "url": null, // no longer needed once https://github.com/microsoft/vscode-pwa/pull/100 is merged
  "server: {
    "command": "npm run electron -- --remote-debugging-port=9222"
  }
}
```
